packages/@aws-cdk/aws-s3/lib/bucket.ts

```diff
@@ -54,6 +54,13 @@ export interface IBucket extends IResource {
    */
   readonly bucketRegionalDomainName: string;
 
+  /**
+   * If this bucket has been configured for static website hosting.
+   *
+   * @default false
```

Comment:
There is no meaning for @default in interfaces, only on structs (interfaces that don't start with an `I`).